website/tests/api_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from website.add_youtuber import YoutubeApi, API_KEY
+
+
+class TestYoutubeApi(unittest.TestCase):
```

Comment:
I'd try to mock it somehow, so your tests succeed (do not make the internet requests actually) while youtube is unavailable.

Reply:
I want to test interaction with YouTube to check YouTube API changes.
If Google makes some changes in YouTube API my tests will fail, in another case with mock it will pass.